src/daemon-node.js

```diff
@@ -122,7 +125,16 @@ class Node {
       this.path = initOpts.directory
     }
 
-    const args = ['init', '-b', initOpts.keysize || 2048]
+    let keysize = initOpts.keysize
+    if (!keysize) {
+      if (typeof global.it === 'function') {
+        keysize = this.opts.type === 'go' ? 1024 : 512
```

Comment:
@richardschneider why are we setting different sizes for go and js, could you please remind me?

Reply:
RTFM the top comment on this PR